main.go

```diff
@@ -266,12 +277,14 @@ func makeHTTPServer(aggregatedClients [][]redis.UniversalClient, mongo *mongo.Cl
 		ctx, cancel := context.WithTimeout(context.Background(), config.MongoConnectTimeout())
 		defer cancel()
 
-		mongoErr := mongo.Ping(ctx, readpref.Primary())
-		mongoOK := mongoErr == nil
-
-		if !mongoOK {
-			log.Log.Errorw("Error connecting to Mongo during healthz check",
-				"error", mongoErr)
+		mongoOK := true
+		for _, mongo := range aggregatedMongos {
+			mongoErr := mongo.Ping(ctx, readpref.Primary())
+			mongoOK := (mongoOK && (mongoErr == nil))
```

Comment:
isn't this shadowing the one that's actually read by the health check endpoint?

Reply:
oh good catch

Reply:
hey! one of my three comments was useful!